### PR TITLE
fluxcd: 0.39.0 -> 0.40.0

### PR DESCRIPTION
**Pull request by superherointj**
fluxcd: 0.39.0 -> 0.40.0

Release: https://github.com/fluxcd/flux2/releases/tag/v0.40.0

### CODE REVIEW

**Comment by matthewpi** [comment]
Result of `nixpkgs-review pr 217672` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>fluxcd</li>
  </ul>
</details>

File: pkgs/applications/networking/cluster/fluxcd/update.sh [hunk 1]
```diff
@@ -38,7 +38,9 @@ if [ ! "$OLD_VERSION" = "$LATEST_VERSION" ]; then
     if [ $# -eq 1 ] && [ "$1" = "git" ]; then
         git switch -c "package-fluxcd-${LATEST_VERSION}"
         git add "$FLUXCD_PATH"/default.nix
-        git commit -m "fluxcd: ${OLD_VERSION} -> ${LATEST_VERSION}"
+        git commit -m "fluxcd: ${OLD_VERSION} -> ${LATEST_VERSION}
+
+Release: https://github.com/fluxcd/flux2/releases/tag/v${LATEST_VERSION}"
```
Comment (bryanasdev000) [inline pkgs/applications/networking/cluster/fluxcd/update.sh:43]: Missing # here to mark as comment?

**Review by bryanasdev000 — CHANGES_REQUESTED** [review]
LGTM, only missing a # to probably mark as comment.

File: pkgs/applications/networking/cluster/fluxcd/update.sh (hunk 1 above)
Comment (bryanasdev000) [reply pkgs/applications/networking/cluster/fluxcd/update.sh:43]: Never mind, it's the commit message :P

**Review by bryanasdev000 — APPROVED** [review]

**Pull request merged**